build.gradle

```diff
@@ -28,36 +22,47 @@ plugins {
 
 apply plugin: 'io.spring.nohttp'
 
-//def gitPresent = new File('.git').exists()
+def gitPresent = new File('.git').exists()
 
-//if(gitPresent) {
-//	apply plugin: 'org.ajoberstar.grgit'
-//}
+if(gitPresent) {
+	apply plugin: 'org.ajoberstar.grgit'
+}
 
 description = 'Spring for Apache Pulsar'
 
 ext {
-//	if (gitPresent) {
-//		modifiedFiles =
-//			files(grgit.status().unstaged.modified).filter{ f -> f.name.endsWith('.java') || f.name.endsWith('.kt') }
-//	}
+	linkHomepage = 'https://github.com/spring-projects-experimental/spring-pulsar'
+	linkIssue = 'https://github.com/spring-projects-experimental/spring-pulsar/issues'
+	linkScmUrl = 'https://github.com/spring-projects-experimental/spring-pulsar'
+	linkScmConnection = 'https://github.com/spring-projects-experimental/spring-pulsar.git'
+	linkScmDevConnection = 'git@github.com:spring-projects-experimental/spring-pulsar.git'
+	docResourcesVersion = '0.2.5'
+
+	javadocLinks = [
+			'https://docs.oracle.com/en/java/javase/17/docs/api/',
+			'https://docs.spring.io/spring-framework/docs/current/javadoc-api/'
+	] as String[]
+
+	if (gitPresent) {
+		modifiedFiles =
+				files(grgit.status().unstaged.modified).filter{ f -> f.name.endsWith('.java') || f.name.endsWith('.kt') }
```

Comment:
Do we have any Kotlin files in here currently? If not, and we don't plan on adding anytime soon, I would be in favor of removing any Kotlin related pieces.

Reply:
will remove.